src/instructlab/configuration.py

```diff
@@ -138,20 +138,39 @@ class _generate(BaseModel):
     seed_file: StrictStr = "seed_tasks.json"
 
 
+class _serve_vllm(BaseModel):
```

Comment:
you need to mansually splice these into the `default_map`. I do something similar for training below.